client/my-sites/email/email-providers-stacked-comparison/provider-cards/provider-card-props.ts

```diff
@@ -0,0 +1,63 @@
+import { Domain } from '@automattic/data-stores/dist/types/site';
+import { TranslateResult } from 'i18n-calypso';
+import { ReactElement } from 'react';
```

Comment:
These should all be types and the `Domain` type should be imported from the top level of the `data-stores` package.
```suggestion
import type { Domain } from '@automattic/data-stores';
import type { TranslateResult } from 'i18n-calypso';
import type { ReactElement } from 'react';
```